test/tests/rackhd20/test_rackhd20_api_decommission_node.py

```diff
@@ -0,0 +1,116 @@
+'''
+Copyright 2016, EMC, Inc.
+
+'''
+
+import fit_path  # NOQA: unused import
+from nosedep import depends
+import random
+import time
+import flogging
+from nose.plugins.attrib import attr
+from common import fit_common
+
+# set up the logging
+log = flogging.get_loggers()
+
+# Local methods
+NODECATALOG = fit_common.node_select()
+
+
+# Decommission base payload
+DECOMMISSION_PAYLOAD = {
+    "name": "Graph.Bootstrap.Decommission.Node",
+    "options": {
+        "defaults": {
+        }
+    }
+}
+
+# validate base payload
+VALIDATE_PAYLOAD = {
+    "name": "Graph.Bootstrap.Decommission.Node.Test",
+    "options": {
+        "defaults": {
+        }
+    }
+}
+
+
+@attr(regression=True, smoke=False)
+class rackhd20_api_workflows(fit_common.unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        # Get the list of nodes
+        NODECATALOG = fit_common.node_select()
+        # Select one node at random
+        cls.__NODE = NODECATALOG[random.randint(0, len(NODECATALOG) - 1)]
+        # delete active workflows for specified node
+        fit_common.cancel_active_workflows(cls.__NODE)
+
+    # this routine polls a workflow task ID for completion
+    def __wait_for_workflow_complete(self, instanceid, start_time, waittime=900, cycle=30):
+        log.info_5(" Workflow started at time: %f", start_time)
+        while time.time() - start_time < waittime:
+            result = fit_common.rackhdapi("/api/2.0/workflows/" + instanceid)
+            if result['status'] != 200:
+                log.error(" HTTP error: " + result['text'])
+                return False
+            if result['json']['status'] in ['running', 'pending']:
+                log.info_5("{} workflow status: {}".format(result['json']['injectableName'], result['json']['status']))
+                fit_common.time.sleep(cycle)
```

Comment:
Please remove the fit_common here.  We should use this directly from the time import.